src/main/java/fi/dy/masa/tweakeroo/renderer/RenderUtils.java

```diff
@@ -270,7 +269,7 @@ public static float getLavaFogDistance(Entity entity, float originalFog)
         {
             final int resp = EnchantmentHelper.getRespiration(living);
             final int aqua = EnchantmentHelper.getEquipmentLevel(Enchantments.AQUA_AFFINITY, living);
-            float fog = originalFog;
+            float fog = (originalFog > 1.0f) ? 3.3f : 1.3f;
```

Comment:
Clamps new Vanilla values to still return roughly the same values as 1.20.4 after Multiplication.